lib/lighthouse/benefits_documents/service.rb

```diff
@@ -19,29 +19,36 @@ def initialize(icn)
 
     def queue_document_upload(params, lighthouse_client_id = nil)
       start_timer = Time.zone.now
-      claim_id = params[:claimId]
-      raise Common::Exceptions::ArgumentError, claim_id unless claim_id
+      claim_id = params[:claimId] || params[:claim_id]
```

Comment:
for both `queue_document_upload` and `queue_multi_image_upload_document` I was running into test issues strictly based on what case the parameters were in. Since there is a conflict between what case ruby is historically written in and what javascript sends, I changed this to accommodate both

Reply:
You have to set up an initializer to do the conversion I believe. If the front end is built using rails helpers though it makes the parameters snake_case so you never have to think about it

Reply:
Looking at the other vets-api code, it seems like we almost always just use the snake_case version, so I think it's safe to assume that params are being converted to snake_case somehow

Reply:
I thought so too but then I saw in other specs some parameters were using camelCase so I wasn't sure

Reply:
@jerekshoe so do you think it's best if I remove this guard for multiple cases

Reply:
I'm curious if @jperk51 has any thoughts on this. I'm all for removing it to simplify the code if possible, but if there is a need to have it there then obviously we would want to leave it as you have it now

Reply:
since camel is how it was before and it was working I'm hesitant to change it. @Tonksthebear go ahead and add a long message under this to see if we are getting camel or snake. We'll check that logging and return to this once we know changing it work break anything and if it's camel we'll figure out why and fix it.

Reply:
@jerekshoe @jperk51 I added logging so that we can keep an eye on what case the parameters come in as. Jerek, we've been bitten in the past with wrong casing of parameters so we'd like to keep it this way for now and we can see how the parameters come in from the app